cmd/githubRemoveValidate.go

```diff
@@ -33,6 +33,14 @@ var githubRemoveValidateCmd = &cobra.Command{
 		if err == nil {
 			fmt.Println("gitops not found as expected")
 		}
+		repoMetaphorGo, err := gitWrapper.GetRepo(owner, "metaphor-go")
+		if err == nil {
+			fmt.Println("gitops not found as expected")
+		}
+		repoMetaphorFrontend, err := gitWrapper.GetRepo(owner, "metaphor-frontend")
+		if err == nil {
```

Comment:
is this logic correct? I believe it should be if err != nil {

Reply:
Same as above.